test/e2e/test_image.py

```diff
@@ -98,13 +98,64 @@ def test_create_with_invalid_image_file(self):
         self.check_post_image_failed_with_detail(422, "File given is not a valid image")
 
 
-# pylint:disable=fixme
-# TODO: Inherit from GetDSL when added
-class ListDSL(CreateDSL):
-    """Base class for list tests."""
+class GetDSL(CreateDSL):
+    """Base class for get tests."""
 
     _get_response_image: Response
 
+    def get_image(self, image_id: str) -> None:
+        """
+        Gets an image with the given ID.
+
+        :param image_id: The ID of the image to be obtained.
+        """
+        self._get_response_image = self.test_client.get(f"/images/{image_id}")
+
+    def check_get_image_success(self, expected_image_data: dict) -> None:
+        """
+        Checks that a prior call to `get_image` gave a successful response with the expected data returned.
+
+        :param expected_image_data: Dictionary containing the expected image data as would be required
+            for an `ImageSchema`.
+        """
+        assert self._get_response_image.status_code == 200
+        response_json = self._get_response_image.json()
+        del response_json["url"]
+        assert response_json == expected_image_data
```

Comment:
If renaming the schemas the IMAGE_GET_DATA_ALL_VALUES an become IMAGE_METDATA_GET_DATA_ALL_VALUES, then you can add a new one
```IMAGE_GET_DATA_ALL_VALUES = {**IMAGE_METDATA_GET_DATA_ALL_VALUES, url: ANY}```
Allowing this to be
```suggestion
        assert self._get_response_image.json() == expected_image_data
```